packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -772,6 +772,26 @@ export const [
     GridHeaderFallbackFontFamily
 );
 
+export const [
+    mentionFont,
+    mentionFontColor,
+    mentionDisabledFontColor,
+    mentionFontFamily,
+    mentionFontWeight,
+    mentionFontSize,
+    mentionFontLineHeight,
+    mentionFallbackFontFamily
+] = createFontTokens(
+    tokenNames.mentionFont,
+    (element: HTMLElement) => getColorForTheme(element, DigitalGreenDark, PowerGreen, PowerGreen),
+    (element: HTMLElement) => hexToRgbaCssColor(getDefaultFontColorForTheme(element), 0.3),
+    BodyFamily,
+    BodyWeight,
```

Comment:
Just want to double check that we want the regular body text weight and not something like `BodyEmphasizedWeight`.

Reply:
Since we have the option to change the text formatting to bold, we thought the text could be in body text weight instead of emphasized text.